operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -350,6 +350,16 @@ alertmanager:
       cpu: 10m
       memory: 32Mi
 
+  # -- fallback config for alertmanager 
+  fallbackConfig: |
+    route:
+      receiver: 'default-receiver'
+      group_wait: 30s
+      group_interval: 5m
+      repeat_interval: 4h
+      group_by: [cluster, alertname]
```

Comment:
This didn't work when I tried setting it as my config. Should it be something that alertmanager accepts by default? I managed to make it work with 

```yaml
    global:
        smtp_smarthost: 'localhost:25'
        smtp_from: 'alertmanager@example.org'
        smtp_auth_username: 'alertmanager'
        smtp_auth_password: 'password'
    route:
        receiver: 'default-receiver'
        group_wait: 30s
        group_interval: 5m
        repeat_interval: 4h
        group_by: [cluster, alertname]
    receivers:
        - name: 'default-receiver'
          email_configs:
          - to: 'alerts@example.org'

```

Reply:
I agree, i've updated the default to  yours.